packages/terminal/src/browser/terminal-linkmatcher-files.ts

```diff
@@ -86,12 +88,12 @@ export class TerminalLinkmatcherFiles extends AbstractCmdClickTerminalContributi
     }
 
     protected async toURI(match: string, cwd: URI): Promise<URI | undefined> {
-        const path = await this.extractPath(match);
-        if (!path) {
+        const extractedPath = await this.extractPath(match);
+        if (!extractedPath) {
             return;
         }
-        const pathObj = new Path(path);
-        return pathObj.isAbsolute ? cwd.withPath(path) : cwd.resolve(path);
+        const pathObj = new Path(extractedPath);
+        return pathObj.isAbsolute ? cwd.withPath(extractedPath) : cwd.resolve(extractedPath);
```

Comment:
This refactor is to avoid `[tslint] Shadowed name: 'path' (no-shadowed-variable)` error, which comes from the introduction of Node.JS path module.